### PR TITLE
Launch Amplify iOS and Android to GA

### PR DESCRIPTION
**Pull request by jpignata**
*Issue #, if available:* n/a

*Description of changes:* Changes across the categories to support the 1.0.0 release of Amplify iOS and Android.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: client/src/docs-ui/choose-anchor/choose-anchor.tsx [hunk 1]
```diff
@@ -35,7 +21,7 @@ export class DocsChooseAnchor {
           {filterKey &&
             Object.entries(filterMetadataByOptionByName[filterKey]).map(
               ([filterValue, {label, graphicURI}]) => {
-                const route = getRoute(this.page, filterValue);
+                const route = this.page.versions?.[filterValue] as string;
```
Comment (sammartinez) [inline client/src/docs-ui/choose-anchor/choose-anchor.tsx:24]: @jordanranz Were you fine with this?

Comment (TrekSoft) [reply client/src/docs-ui/choose-anchor/choose-anchor.tsx:24]: You have about 23 seconds to answer

Comment (jpignata) [reply client/src/docs-ui/choose-anchor/choose-anchor.tsx:24]: Yes, @jordanranz approved this pull.

Comment (jordanranz) [reply client/src/docs-ui/choose-anchor/choose-anchor.tsx:24]: yes

**Pull request merged**